contrib/vagrant/provision-util.sh

```diff
@@ -128,6 +128,9 @@ os::provision::set-os-env() {
   fi
   os::provision::set-bash-env "${origin_root}" "${config_root}" \
 "/root/${file_target}"
+
+  # Make symlinks to the bash completions for the openshift commands
+  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
```

Comment:
In that case, lgtm.

It seems strange to see you not rewriting your history, but it's saner in a way, too.